src/wishlist.tm

```diff
@@ -21,7 +21,55 @@
 
   \ <hrule>
 
-  \;
+  Sometimes, one has to produce quite compacts documents, where space is key
+  (e.g.<nbsp>a short report, an exercice sheet or correction<text-dots>). One
+  very important feature you be to put figures on the left or the right of
+  the page, in a floating manner. To emulate this, tables can be used, with
+  one cell for the text and one cell for the figure, but it does not work
+  well at all with figure legends, footnotes<text-dots>
+
+  More generally, mouse-movable floating minipages would be a killer feature
+  and solve many issues at the same time.
+
+  <hrule>
+
+  An user-friendly interface to create/modify shortcuts.
+
+  <hrule>
+
+  Being able to easily create macros, at least non-parametrical ones, with a
+  single menu action. It is already possible, but the syntax not easy for
+  begginers (and non-programmers in general).
+
+  <hrule>
+
+  Being able to adjust the math typesetting parameters. For example, by
+  default, one may find formulas too compact, and would like to have more
+  space around (in)equality signs.
+
+  <hrule>
+
+  A variant for underbraces and overbraces with <verbatim|\\smash> ;
+  currently <verbatim|\<less\>resize\<gtr\>> can be used but it is slow to
+  use and content can't be really modified afterwards.
+
+  <hrule>
+
+  A set of shortcuts for common colors (red, dark green, blue, grey).
+
+  <hrule>
+
+  A set of shotcuts for commonly used unicode symbols in text mode, such as
+  \<Rightarrow\>, \<rightarrow\>, <text-dots> similar to shortcuts that exist
+  in math mode.
+
+  <hrule>
+
+  The "draw image" feature is heavily bugged. It may be wiser to not loose
```

Comment:
Please avoid these subjective statement. Many people use it and albeit not perfect it is certainly not "heavily bugged" Please read the tutorial on the graphics mode on the TeXmacs webpage, the design of the UI is not the most common.

Reply:
Yes indeed. I did read the documentation, and I really meant bug, including crashes and document corruption. Very frequently. But that's a very minor point, and I deleted it.

Reply:
I use it too (e.g. https://www.youtube.com/watch?v=Au1CpFS5VCU&t=28s) and it seems to me to work fine. I never experienced serious problems in recent versions. I also find that the interface is awkward, but actually when you get a bit used you can do non-trivial things. It is not meant to be a replacement for Inkscape, it is meant to be an easy to use too for small work. It would be useful if you can post the bugs in the bugtracker.